Clean-up/test_clean_lambda.py

```diff
@@ -0,0 +1,54 @@
+"""This file tests whether the clean_lambda file functions correctly"""
+
+import json
+from unittest.mock import patch, MagicMock
+
+import pytest
+
+from clean_lambda import delete_unsubscribed, handler
+
+
+@pytest.fixture(name="unsubscribed_products")
+def fixture_unsubscribed_products() -> list[dict]:
```

Comment:
Nice, I really like the use of name in the fixture decorator. I'm going to start doing this.

Reply:
Thank you